README.md

```diff
@@ -3,47 +3,52 @@
 [EIP2771-compatible](https://eips.ethereum.org/EIPS/eip-2771) transaction relayer library for Ethereum. The core of the library is bindings to a compatible forwarder contract. 
 
 Components:
-- `cmd`: example CLI that uses examples/minimal_forwarder
-- `examples`: example implementations of forwarder and recipient contracts, as well as Go bindings, required interface implementations, and unit tests
-	- `examples/gsn_forwarder`: an implementation using OpenGSN's [forwarder contract](https://github.com/opengsn/gsn/tree/master/packages/contracts/src/forwarder).
-	- `examples/minimal_forwarder`: an implementation using OpenZeppelin's [minimal forwarder](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/MinimalForwarder.sol).
-	- `examples/mock_recipient`: an implementation of a EIP2771-compatible recipient contract which can receive calls from a forwarder.
+- `cmd/relayer`: example CLI that uses impls/minimal_forwarder
+- `impls`: example implementations of forwarder and recipient contracts, as well as Go
+  bindings, required interface implementations, and unit tests
+	- `impls/gsn_forwarder`: an implementation using OpenGSN's [forwarder contract](https://github.com/opengsn/gsn/tree/master/packages/contracts/src/forwarder).
+	- `impls/minimal_forwarder`: an implementation using OpenZeppelin's [minimal forwarder](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/MinimalForwarder.sol).
+- `examples/mock_recipient`: an implementation of a EIP2771-compatible recipient contract
```

Comment:
I couldn't keep this one indented under the same `impls` bullet because it never moved out of the `examples` folder like the other ones.